### PR TITLE
Menu scene

### LINKED ISSUE #78: Basic menu scene with level selection
- [x] start menu scene
- [x] level selection scene

### PR DESCRIPTION
**Pull request by Castelias**
closes #78

### CODE REVIEW

File: Assets/Scripts/Player/Player.cs [hunk 1]
```diff
@@ -7,7 +7,9 @@
 public class Player : MonoBehaviour
 {
     public Text showCoinsAmount;
-    private int coins=0;
+    public Text showKillsAmount;
+    private int _coins=0;
+    private int _kills;
```
Comment (PeterPwlk) [inline Assets/Scripts/Player/Player.cs:12]: Why is kills not initialized and coins is?

File: Assets/Scripts/Player/Player.cs [hunk 2]
```diff
@@ -16,7 +18,8 @@ private void Start()
 
     void Update()
     {
-        showCoinsAmount.text = coins.ToString();
+        showCoinsAmount.text = _coins.ToString();
+        showKillsAmount.text = _kills.ToString();
```
Comment (PeterPwlk) [inline Assets/Scripts/Player/Player.cs:22]: Calling it in Update is a bad idea. It should be called only if the value changes.

File: Assets/Scripts/UI/ChoseScene.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class ChoseScene : MonoBehaviour
+{
+    public void MainScene()
+    {
```
Comment (PeterPwlk) [inline Assets/Scripts/UI/ChoseScene.cs:9]: All those methods could be merged into one that takes a scene name as a parameter

File: Assets/Scripts/UI/MaineMenu.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+public class MaineMenu : MonoBehaviour
+{
```
Comment (PeterPwlk) [inline Assets/Scripts/UI/MaineMenu.cs:8]: Typo in class name

File: Assets/Scripts/UI/ChoseScene.cs (hunk 3 above)
Comment (Castelias) [reply Assets/Scripts/UI/ChoseScene.cs:9]: Nah it can't!

File: Assets/Scripts/Player/Player.cs (hunk 1 above)
Comment (piskorzm) [reply Assets/Scripts/Player/Player.cs:12]: https://www.tutorialspoint.com/csharp/csharp_data_types.htm#:~:text=ValueType.,memory%20to%20store%20the%20value. This says an int value already has a default value of zero. Try it

Comment (PeterPwlk) [reply Assets/Scripts/Player/Player.cs:12]: If it has a default of 0, coins initialization can be removed :D

**Pull request merged**